### ISSUE TITLE
@docusaurus/core and @docusaurus/theme-common are out of date

### ISSUE THREAD

**Issue by Zamiell** [typesense/docusaurus-theme-search-typesense#39]
I get the following errors when installing this library:

```
➤ YN0060: │ @docusaurus/core is listed by your project with version 3.0.1, which doesn't satisfy what docusaurus-theme-search-typesense (pe3d48) requests (3.0.0).
➤ YN0060: │ @docusaurus/theme-common is listed by your project with version 3.0.1, which doesn't satisfy what docusaurus-theme-search-typesense (pf65e6) requests (3.0.0).
```

Can these be updated? Additionally, maybe the ranges should be less strict to allow for any version in the range between 3.0.0 >= x < 4.0.0 ?

**Comment by jasonbosco** [comment]
@Zamiell Could you try with `v0.14.1-0` of `docusaurus-theme-search-typesense`? 

> Additionally, maybe the ranges should be less strict to allow for any version in the range between 3.0.0 >= x < 4.0.0 ?

In the past, I've seen Docusaurus crash if these version numbers don't exactly match... which is why I pinned them to track the docusaurus versions.

**Comment by Zamiell** [comment]
@jasonbosco `v0.14.1-0` seems to work great, thanks. Can you push it live?

**Comment by jasonbosco** [comment]
Thank you for confirming. Just published `0.14.1`.
